lib/facter/virtual.rb

```diff
@@ -90,6 +90,17 @@
 end
 
 Facter.add("virtual") do
+  confine :kernel => 'SunOS'
```

Comment:
Was this necessary, or just re-organization to put it with other definitions of virtual?